modules/olm-configuring-aws-sts.adoc

```diff
@@ -0,0 +1,182 @@
+
+Some `ccoctl` commands make AWS API calls to create or modify AWS resources. You can use the `--dry-run` flag to avoid making API calls. Using this flag creates JSON files on the local file system instead. You can review and modify the JSON files and then apply them with the AWS CLI tool using the `--cli-input-json` parameters.
+====
+
+.Prerequisites
+
+* You have an AWS cluster that is configured to use STS.
+
+* You have installed an Operator that can use STS.
+
+.Procedure
+
+. Configure the `ccoctl` utility:
+
+.. Get the CCO container image from the {product-title} release image by running the following command:
++
+[source,terminal]
+----
+$ CCO_IMAGE=$(oc adm release info --image-for='cloud-credential-operator')
+----
+
+.. Extract the `ccoctl` binary from the CCO container image within the {product-title} release image by running the following command:
++
+[source,terminal]
+----
+$ oc image extract $CCO_IMAGE --file="/usr/bin/ccoctl" -a ${PULL_SECRET_FILE}
+----
+//Need to explain/define PULL_SECRET_FILE I think
+
+.. Change the permissions to make `ccoctl` executable by running the following command:
++
+[source,terminal]
+----
+$ chmod 775 ccoctl
+----
+////
+//This strat may not actually make sense due to how STS is implemented and given the move away from minting.
+ifdef::known-operator[]
+. Determine the name of the `CredentialsRequest` custom resource (CR) in the `openshift-cloud-credential-operator` namespace for the Operator: #How do they do that?#
+
+. Extract the CR to the `credrequests` directory by running the following command:
++
+[source,terminal]
+----
+$ oc get credentialsrequest \
+  -n openshift-cloud-credential-operator <custom_resource_name> \
+  -o yaml > <path_to_directory_with_list_of_credentials_requests>/credrequests/<custom_resource_name>.yaml
+----
+endif::[]
+////
+//ifdef::olm-generic[]
+. Create and save a `CredentialsRequest` custom resource (CR) YAML file, such as shown in the following example, and then place it in the `credrequests` directory:
++
+.Example
++
+[source, yaml]
+----
+apiVersion: cloudcredential.openshift.io/v1
+kind: CredentialsRequest
+metadata:
+  name: openshift-aws-efs-csi-driver <1>
+  namespace: openshift-cloud-credential-operator <2>
+spec:
+  providerSpec:
+    apiVersion: cloudcredential.openshift.io/v1
+    kind: AWSProviderSpec
+    statementEntries: <3>
+    - action:
+      - elasticfilesystem:*
+      effect: Allow
+      resource: '*'
+  secretRef: 
+    name: aws-efs-cloud-credentials <4>
+    namespace: openshift-cluster-csi-drivers <5>
+  serviceAccountNames: <6>
+  - aws-efs-csi-driver-operator
+  - aws-efs-csi-driver-controller-sa
+----
+<1> Specify the name of the `CredentialsRequest` CR. (#user discretion for Operators that don't provide one?#)
+<2> Specify the `openshift-cloud-credential-operator` namespace.
+<3> #needinfo here#
```

Comment:
I commented [here](https://github.com/openshift/openshift-docs/pull/51541#discussion_r992800823). You can reference the AWS Docs [IAM Policy Document format (regular format)](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_elements_statement.html) to guide the users to build their own policies.